### PR TITLE
Update wasm binary format label to 1

### PR DESCRIPTION
**Pull request by dschuff**
Rebaseline tests, but no other changes.

### CODE REVIEW

**Comment by dschuff** [comment]
The spec tests and interpeter still use 0xd, so we may have to disable the spec tests or do some other hackery to keep them working?

**Comment by sbc100** [comment]
Can we get the spec tests and interpreter updated?

**Comment by binji** [comment]
Yeah, I'd rather update the spec tests.

**Comment by binji** [comment]
OK, I've landed the update to v1 in the spec repo, and updated github.com/WebAssembly/testsuite. You should be able to just update wabt's testsuite submodule to latest to fix this.

File: CMakeLists.txt [hunk 1]
```diff
@@ -89,9 +89,15 @@ else ()
   #   interfaces, etc.
   # disable -Wpointer-arith: this is a GCC extension, and doesn't work in MSVC.
   add_definitions(
-    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g
+    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g -std=c++11
```
Comment (jfbastien) [inline CMakeLists.txt]: y u no C++14?

Comment (binji) [inline CMakeLists.txt]: Well, in the past few days we've switched from C89 to C++98 to C++11. I figure by next week we'll be at C++++10k

Comment (dschuff) [inline CMakeLists.txt]: Real question though, have we decided that we are willing to give up VS2013 support yet? And, how is VS2015's C++14 support? Also I wonder which versions of gcc we are willing to support?

Comment (dschuff) [inline CMakeLists.txt]: Partial answers:
1. IMO Yes.
2. https://msdn.microsoft.com/en-us/library/hh567368.aspx
3. Ubuntu Xenial LTS has gcc 5.3, so maybe that?

Comment (binji) [inline CMakeLists.txt]: 1. We don't build w/ appveyor on VS2013, so it's probably broken anyway.
3. I don't really want to move to anything later than 4.8.4 since that's what I have on my laptop. :)

Comment (dschuff) [inline CMakeLists.txt]: I sympathize, but our machines are pretty outdated at this point (coming up on a 3-year-old distro of a 4-year-old compiler branch, and replacement LTS has been out for nearly a year now). It seems unlikely we'd make such concessions for Windows or Mac users, so why should we do that for ourselves? It's really easy to build a daily-driver clang or gcc (libc++ is a little harder, but still not that hard).

Comment (binji) [inline CMakeLists.txt]: Yeah, I suppose so. Thought it looks like Travis is also at gcc 4.8.4 and clang 3.5 by default, so we'll have to update those too. Anyway, probably worth moving this convo to an actual issue. :)

Comment (jfbastien) [inline CMakeLists.txt]: Guys, you realize this was a drive-by-troll, right? You've made it to Horse wasm, so I guess that's nice.

Comment (dschuff) [inline CMakeLists.txt]: I totally do, but it's a legitimate question to ask again now that we're going to C++

Comment (dschuff) [inline CMakeLists.txt]: and yes, to an actual issue :)
